f8a_worker/workers/CVEchecker.py

```diff
@@ -19,16 +21,46 @@ class CVEcheckerTask(BaseTask):
     schema_ref = SchemaRef(_analysis_name, '3-0-0')
 
     @staticmethod
+    def get_cve_impact(id_):
+        score = 0
+        vector = ''
+        severity = ''
+        if id_:
+            url = "https://nvd.nist.gov/vuln/detail/{id_}".format(id_=id_)
```

Comment:
This is just nitpicking, but you don't have to define variable here and put into requests.get instead